### PR TITLE
Added client removal capabilities with confirmation dialog

### PR DESCRIPTION
**Pull request by JuanNRamirez**

### CODE REVIEW

File: react-app/src/components/RemoveClientModal/RemoveClientModal.tsx [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+export default function RemoveClientModal( {deleteClients}: any ) {
+    const [open, setOpen] = React.useState(false);
+    const handleOpen = () => setOpen(true);
+    const handleClose = () => setOpen(false);
+    const handleYes = () => { deleteClients(); setOpen(false); }
+
+    return (
+        <div>
+            <Button variant="contained" onClick={handleOpen}>
+                Delete Selected Teams
```
Comment (lizcooper44) [inline react-app/src/components/RemoveClientModal/RemoveClientModal.tsx:28]: Might make more sense to have this button say 'Delete Selected Clients'

**Review by lizcooper44 — CHANGES_REQUESTED** [review]
Other than changing button name this LGTM. Nice work with the popup! Eventually we can add some styling to it but this is great functionality

File: react-app/src/components/RemoveClientModal/RemoveClientModal.tsx (hunk 1 above)
Comment (JuanNRamirez) [reply react-app/src/components/RemoveClientModal/RemoveClientModal.tsx]: Good catch! Fixed.

**Review by Pratheekl22 — APPROVED** [review]
LGTM

**Review by lizcooper44 — APPROVED** [review]
LGTM

**Pull request merged**